docker/Dockerfile-base

```diff
@@ -30,23 +40,19 @@ RUN wget --quiet https://repo.anaconda.com/miniconda/Miniconda3-py38_4.10.3-Linu
 ENV PATH=/opt/conda/bin:$PATH
 
 # conda installations ordered by channel
+
+# TODO Figure out why we need to support this and `armory-base-requirements` separately.
+
+# Conda Install for Channel: pytorch
 RUN /opt/conda/bin/conda install \
     pytorch=1.10 \
     torchvision \
     torchaudio \
     cudatoolkit=11.3 \
     -c pytorch \
     && /opt/conda/bin/conda clean --all
-# pytorch==1.10.2 \
-# torchvision==0.11.3 \
-# torchaudio==0.10.2 \
-# cudatoolkit==11.3.1 \
-# ffmpeg==4.3 \
-# numpy==1.21.2 \
-# pillow==8.4.0 \
-# tqdm==4.61.2 \
-# requests==2.25.1 \
```

Comment:
We should either keep these 5 as comments or add them to the conda install without (or with minimal) version info, since they are used directly by our code. That will make it clear how they are installed.